interpreter/terminal_interface/utils/get_config.py

```diff
@@ -47,9 +47,18 @@ def get_config_path(path=user_config_path):
 
 def get_config(path=user_config_path):
     path = get_config_path(path)
+    
+    config = None
 
     with open(path, "r") as file:
         config = yaml.safe_load(file)
-        if config is None:
-            config = {}
-        return config
+        if not config is None:
```

Comment:
`if config is not None:` is better syntax
![image](https://github.com/KillianLucas/open-interpreter/assets/4324290/89555cba-f76e-4d7b-afb1-c75430ed0bd1)
from: https://pep8.org/